src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -594,6 +599,20 @@ public FeedSource clone () throws CloneNotSupportedException {
         return (FeedSource) super.clone();
     }
 
+    /**
+     * Check if there are active jobs to fetch or process a new version for this feed source. This is helpful in the
+     * context of auto-deploying to OTP to determine if there are any jobs occurring that could result in a new feed
+     * version being created (we want to throttle auto-deployments if multiple jobs to create new feed versions are
+     * occurring at the same time).
+     */
+    public boolean hasJobsInProgress() {
+        return StatusController
+            .filterActiveJobs(StatusController.getAllJobs())
+            .stream()
+            .filter(job -> job instanceof FeedSourceJob)
```

Comment:
I'm a little wary of this generalized `instanceOf` check casting a wider net than we want at some point in the future (e.g., on a job that does some (non-fetching/loading) thing to a `FeedSource` like uploading the archived zip files to some external resource. That kind of job would make sense as a `FeedSourceJob`, but we don't really care for the purposes of this method whether it is in progress because it wouldn't impact the AutoDeploy stuff.

Reply:
Reverted back to a more specific job type check.